### DISCUSSION TITLE
Commercial Support Discussion

### DISCUSSION THREAD

**Discussion by tahseenjamal** [btcpayserver/btcpayserver#3126] [Technical Support] [answered]
If I need commercial support for

1. Deployment from security / scalability perspective
2. New features as per requirement
3. Integrations to any other system / app / monitoring / reporting

Who should I connect with ?

**Comment by pavlenex** [comment, marked as answer]
Some community members provide paid support, https://docs.btcpayserver.org/Support/#paid-support

BTCPay as a project doesn't provide commercial support.
